### ISSUE TITLE
SVSGAN implementation

### ISSUE THREAD

**Issue by ShengleiH** [nameless-Chatoyant/singing_voice_separation-pytorch#1]
Hello, have you implemented SVSGAN now? I am implement it recently, but in my model, no matter how I tune the learning rate, the generator's loss is always increasing, and finally the discriminator's loss becomes really small. Can you give me some ideas. Thank you a lot ~
